src/queries/chainflip/assets.ts

```diff
@@ -10,27 +10,37 @@ import {
   usdcOnSolanaAssetId,
   usdtAssetId,
 } from 'constants/caip'
+import mirror from 'lodash/invert'
 
 import { reactQueries } from '../react-queries'
 import type { ChainflipAssetsResponse } from './types'
 
-// Map Chainflip internal asset IDs to CAIPs
-const chainflipToAssetId: Record<string, string> = {
-  'btc.btc': btcAssetId,
-  'eth.arb': arbitrumAssetId,
-  'eth.eth': ethAssetId,
-  'flip.eth': flipAssetId,
-  'sol.sol': solAssetId,
-  'usdc.arb': usdcOnArbitrumOneAssetId,
-  'usdc.eth': usdcAssetId,
-  'usdc.sol': usdcOnSolanaAssetId,
-  'usdt.eth': usdtAssetId,
+// Map CAIPs to Chainflip internal asset IDs
+const assetIdToChainflip: Record<string, string> = {
```

Comment:
Did we want to use our nominal CAIP typing?

```suggestion
const assetIdToChainflip: Record<AssetId, string> = {
```

Reply:
I don't really know if we want to worry with the nominal CAIP stuff here, but leaving here as a nitpick anyway.

Reply:
100% doesn't hurt to! 03c6d5d